project1/application.py

```diff
@@ -0,0 +1,70 @@
+import os
+import sys
+import time
+
+from flask import Flask, render_template, request, url_for,flash,session
+from flask_session import Session
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import create_engine, desc
+from register import *
+from sqlalchemy.orm import scoped_session, sessionmaker
+from datetime import datetime
+
+
+app = Flask(__name__)
+
+# Check for environment variable
+if not os.getenv("DATABASE_URL"):
+    raise RuntimeError("DATABASE_URL is not set")
+
+
+
+# Configure session to use filesystem
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config["SESSION_PERMANENT"] = False
+app.config["SESSION_TYPE"] = "filesystem"
+engine = create_engine(os.getenv("DATABASE_URL"))
+Session(app)
+db.init_app(app)
+
+with app.app_context():
+	db.create_all()
+
+
+
+@app.route("/register")
+def register():
+	return render_template("registration.html")
+
+
+@app.route("/admin")
+def allusers():
+    user = User.query.order_by(User.Time_registered).all()
+    for i in user:
+    	print(i.Username,i.Password,i.Time_registered)
+
+    return render_template("admin.html", users=user)
+
+
+@app.route("/userDetails", methods=["POST", "GET"])
+def userDetails():
+	db.create_all()
+	if request.method == "POST":
+		username = request.form.get("uname")
+		email = request.form.get("email")
+		gender = request.form.get("gender")
+		password = request.form.get("pwd")
```

Comment:
Try encrypting the password if the time permits as it is a good practice

Reply:
encryption implemented.